dev_requirements.txt

```diff
@@ -3,7 +3,9 @@ Babel==2.12.1
 betterproto==1.2.5
 certifi==2023.7.22
 charset-normalizer==3.1.0
-dbt-tests-adapter==1.7.0
+dbt-core==1.8.8
+dbt-tests-adapter==1.8.0
```

Comment:
[nit]
I see that on `setup.py` we're requiring `dbt-adapters` and on pip `dbt-tests-adapter` links to `dbt-adapters` repository. Has this been deprecated/discontinued? Should we update this?

Reply:
dbt-test-adapters and dbt-adapters - are 2 different libs. 
- test-adapters needed explicitly cause 1.8 migration splited tests from core and now I added it explicitly.
- dbt-adapters is a transitive dbt-core dependency. That's why I'm not mentioning it here intentionally.